shim.js

```diff
@@ -1,3 +1,5 @@
+import 'react-native-get-random-values';
+import '@ethersproject/shims';
```

Comment:
This might remove the need for our rn-nodeify stuff as it polyfills everything

Reply:
got it! thanks